kivymd/uix/button/button.py

```diff
@@ -611,24 +651,23 @@ class BaseButton(
     and defaults to `14sp`.
     """
 
-    user_font_size = NumericProperty(0)
```

Comment:
This will entail a bunch of mistakes. And for many users and in the demo application! You cannot just remove this attribute. You should marked it as deprecated. And make a `user_font_size` method that passes its value to the `icon_size` attribute.

Reply:
It's not gone, just moved; the order of attributes was rearranged slightly when then icon properties were added. In fact I haven't done anything with user_font_size which (I think) was only used with MDIconButton.

But yes, it should be deprecated and replaced with icon_size.